images/metalk8s-alert-logger/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.16.0-alpine AS builder
```

Comment:
Go and Alpine versions should likely be exposed as build arguments, don't you think?

Reply:
Hmm, yes, why not, I'm not sure if it will useful, but it does not cost anything to do it.
I'll change it.

Reply:
Done